packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -0,0 +1,51 @@
+@forward 'settings';
+
+@use 'usa-file-input/src/styles/usa-file-input';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-hint/src/styles/usa-hint';
+@import '../../global/formation_overrides';
+
+/* Original Styles */
+@import '../../mixins/accessibility.css';
+@import '../../mixins/form-field-error.css';
+@import '../../mixins/hint-text.css';
+
+:host {
+  display: block;
+  font-family: var(--font-source-sans);
+}
+
+#error-message {
+  margin-bottom: 0.3rem;
+}
+
+va-button {
+  margin-bottom: -0.8rem;
+}
+
+.required {
+  color: var(--color-secondary-dark);
+  margin-left: 0.4rem;
+}
+
+/* V3/USWDS Styles */
+.usa-label--required,
+.usa-error-message {
+  color: var(--v3-color-error-dark);
+}
+
+:host([error]:not([error=''][uswds]:not([uswds='']))) {
+  border-left-color: var(--v3-color-error-dark);
+}
+
+.usa-file-input .usa-file-input__input[type] {
+  padding: 0;
+}
+
+.usa-file-input__choose {
+  color: var(--color-link-default);
+}
```

Comment:
Why do we need to override the USWDS color here?

Reply:
It was the wrong color blue when I tested it.

Reply:
This is a different color?

--color-link-default is #005ea2 which is rgb(0, 94, 162)

![Screenshot 2024-01-18 at 1 48 29 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/310bec1d-612f-46f5-9c15-7f5aabc6c8b3)